presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/Memo.java

```diff
@@ -243,6 +255,8 @@ static Group withMember(PlanNode member)
         private Multiset<Integer> incomingReferences = HashMultiset.create();
         @Nullable
         private PlanNodeStatsEstimate stats;
+        @Nullable
+        private PlanNodeCostEstimate cumulativeCost;
```

Comment:
Please add a `TODO` comment that this should be stored per plan node. This is only correct for now when memo store single plan node per group.

Reply:
It's obvious that `Memo` needs revamp when groups become real groups.